src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItem.js

```diff
@@ -1,60 +1,111 @@
-      <List.Item key={item.pid}>
-        <List.Icon name="barcode" size="large" verticalAlign="middle" />
-        <List.Content>
-          <Grid columns={3}>
-            <Grid.Column width={5}>
-              <List.Header>{item.barcode}</List.Header>
-              <List.Description>
-                <label>Shelf</label> {item.shelf} <br />
-                <label>Location</label> {item.internal_location.name}
-              </List.Description>
-            </Grid.Column>
-
-            <Grid.Column width={5}>
-              <label>Status</label>{' '}
-              {invenioConfig.ITEMS.canCirculateStatuses.includes(item.status) &&
-              !item.circulation ? (
-                <span className="success">On shelf</span>
-              ) : invenioConfig.ITEMS.canCirculateStatuses.includes(
-                  item.status
-                ) && item.circulation ? (
-                <span className="danger"> On loan </span>
-              ) : (
-                getDisplayVal('ITEMS.statuses', item.status)
-              )}
-            </Grid.Column>
-
-            <Grid.Column width={6}>
-              <List.Description>
-                <label>Medium</label>{' '}
-                {getDisplayVal('ITEMS.mediums', item.medium)}
-                <br />
-                <label>Restrictions</label>{' '}
-                {getDisplayVal(
-                  'ITEMS.circulationRestrictions',
-                  item.circulation_restriction
-                )}
-              </List.Description>
-            </Grid.Column>
-          </Grid>
-        </List.Content>
-      </List.Item>
+      <div className="document-item">
+        {showTitle && (
+          <h3 className="document-item-internal-title">
+            {internalLocationName}
+          </h3>
+        )}
+
+        <Card key={internalLocationName} fluid className="document-item-card">
+          <Table stackable striped compact fixed>
+            <Table.Header>
+              <Table.Row data-test="header">
+                <Table.HeaderCell>Barcode</Table.HeaderCell>
+                <Table.HeaderCell>Shelf</Table.HeaderCell>
+                <Table.HeaderCell>Status</Table.HeaderCell>
+                <Table.HeaderCell>Medium</Table.HeaderCell>
+                <Table.HeaderCell>Restriction</Table.HeaderCell>
+              </Table.Row>
+            </Table.Header>
+
+            <Table.Body>
+              <DocumentItemBody items={itemsToShow} />
+            </Table.Body>
+
+            {this.moreItemsToLoad && (
+              <Table.Footer
+                fullWidth
+                data-test="footer"
+                className="document-item-footer"
+              >
+                <Table.Row>
+                  <Table.HeaderCell colSpan={5} textAlign="right">
+                    <div className="document-item-footer-innerWrapper">
+                      <p className="document-item-footer-text">
+                        Showing entries 1-{itemsToShow.length} of {items.length}{' '}
```

Comment:
Maybe an extra space is nicer?
```suggestion
                        Showing entries 1 - {itemsToShow.length} of {items.length}{' '}
```
If it is without the space in other places, leave it without. :)

Reply:
no spaces in other places 😃